packages/react-vapor/src/components/popover/examples/PopoverConnectedExamples.tsx

```diff
@@ -11,3 +11,7 @@ export const PopoverConnectedExamples = () => (
         </PopoverConnected>
     </div>
 );
+
+export const PopoverExample: ExampleComponent = PopoverConnectedExamples;

```

Comment:
Example files must import only one component :grimacing: 
So you can just remove export on [line 5](https://github.com/coveo/react-vapor/pull/1278/files#diff-6f917e48ba25fcbda1f6b4ef361f2031R5)